src/plugins/projectexplorer/abi.cpp

```diff
@@ -433,6 +451,9 @@ static Abis abiOf(const QByteArray &data)
         if (getUint8(data, pePos) == 'P' && getUint8(data, pePos + 1) == 'E'
             && getUint8(data, pePos + 2) == 0 && getUint8(data, pePos + 3) == 0)
             result = parseCoffHeader(data.mid(pePos + 4));
+
+        }
+
```

Comment:
And one more.

Reply:
I like empty lines around code blocks. :-)